### DISCUSSION TITLE
TMS-90078: Missing Push Notification Entitlement

### DISCUSSION THREAD

**Discussion by gkasireddy202** [expo/expo#31276] [Archived: Questions]
I am getting a warning message from App Store Connect.I did not use the push notification in my app.
ITMS-90078: Missing Push Notification Entitlement - Your app appears to register with the Apple Push Notification service, but the app signature's entitlements do not include the 'aps-environment' entitlement. If your app uses the Apple Push Notification service, make sure your App ID is enabled for Push Notification in the Provisioning Portal, and resubmit after signing your app with a Distribution provisioning profile that includes the 'aps-environment' entitlement. Xcode does not automatically copy the aps-environment entitlement from provisioning profiles at build time. This behavior is intentional. To use this entitlement, either enable Push Notifications in the project editor's Capabilities pane, or manually add the entitlement to your entitlements file. For more information, see

react-native:0.73.7
iOS:17
expo: "50.0.17",
expo-local-authentication: "13.8.0",

**Comment by ngdbao** [comment]
OMG, my app doesn't use notification too. I cannot find out any solution to disable APN, or resolve the warning
